usaspending_api/awards/v2/filters/sub_award.py

```diff
@@ -171,7 +164,7 @@ def recip_string_parse(recipient_string):
             filter_obj = Q()
             for recipient in value:
                 filter_obj |= recip_string_parse(recipient)
-            queryset &= SubawardView.objects.filter(filter_obj)
```

Comment:
Was this structure creating duplicates? Definitely not questioning the choice to make this change, makes much more sense this way, just curious...

Reply:
Nothing is changing in the final queryset object since this queryset is always on the `SubawardView` model. The changes are to clean up the code and to match other querysets which use Q objects and modify the existing queryset